papers/bcr-2020-015-account.md

```diff
@@ -0,0 +1,354 @@
+
+The standard defines 5 levels in a [BIP32] defined derivation path (later expanded to 6 for some multisig script types)
+
+```
+m / purpose' / coin_type' / account' / change / address_index
+``` 
+
+The first two levels define the script type used and the coin/network.
+The next level defines a zero-based index for the account. 
+The penultimate level is either zero or one (receive or change addresses respectively), while the final level is a zero-based index for the keys from which to derive the address. 
+
+When sharing details of different wallets (for example to display addresses or create PSBTs), it is usual to export the following details:
+ - Script type
+ - Fingerprint for the master public key (see [BIP32])
+ - Derivation path from the master key to the account level
+ - The xpub at that derivation
+
+This information can be contained in an output descriptor [OD-IN-CORE], which is defined in CBOR with [BCR-2020-010] and represented in UR using the ``ur:crypto-output`` type. 
+Using this format, each output descriptor defines a single script type, derivation path and parent fingerprint (``crypto-keypath``) and deterministic key (``crypto-hdkey``).
+
+There is a common need however to share information for multiple script types. 
+For example, a user may define the script type on wallet software for which the relevant xpub must be retrieved from a hardware wallet.
+This suggests that a number of output descriptors for standardized script types could be packaged into a CBOR data format.
+Doing so removes the need for the user to also correctly define the matching script type when exporting from the hardware wallet.
+The user would only (optionally) need to specify the account number.
+In addition, this would promote standardization in derivation paths between different wallets.
+
+This document specifies a native CBOR encoding for a number of standardized script types along with their common derivations to the account level. 
+This encoding can be shared as a Uniform Resource [UR] type `crypto-account` (CBOR tag #6.311) for transmitting account level data.
+
+In the encoding, each script type is represented in an output descriptor. 
+These output descriptors are then packaged into an array. 
+Note that in order to include the parent fingerprint for each ``crypto-hdkey``, the master fingerprint (which is common to all accounts) is included separately.
+The top level packaging is a map of the master fingerprint, and the array of output descriptors.
+
+The following standardized script types may be present in a ``crypto-account`` encoding, shown here with default derivations for Bitcoin mainnet and account #0:
+
+| Script type | Default Derivation |
+| ----------- | ---------- |
+| P2PKH | ``m/44'/0'/0'`` |
+| P2SH-P2WPKH | ``m/49'/0'/0'`` |
+| P2WPKH | ``m/84'/0'/0'`` |
+| Multisig P2SH | ``m/45'/0'/0'`` |
```

Comment:
I think it's just `m/45'` here, all other indexes are non-hardened according to bip45.

Reply:
Thanks, corrected and added an explanatory note referencing bip45.